docs/2021-OpenYurt-annual.md

```diff
@@ -0,0 +1,101 @@
+# OpenYurt Sandbox Annual Review
+
+## Background
+
+[OpenYurt](https://openyurt.io/en-us/) is an extension of the upstream Kubernetes for edge computing use cases.
+It has been designed to meet various DevOps requirements against typical edge infrastructures. With OpenYurt,
+users are able to manage their edge applications as if they were running in the cloud infrastructure.
+It addresses specific challenges for cloud-edge orchestration in Kubernetes such as unreliable or disconnected
+cloud-edge networking, edge node autonomy, edge device management, region-aware deployment and so on.
+Currently, OpenYurt primarily provides the following capabilities:
+- A node daemon which proxies the network traffics from Kubelet and/or other daemons so that node components still
+  function even if the node is disconnected to the cloud.
+- A tunnel service to allow the cloud site control plane to access the node components running in the edge nodes.
+- A supplement to upstream controllers to deal with edge failures.
+- A workload for managing edge applications across multiple edge regions.
+- An integration with [EdgeX Foundry](https://github.com/edgexfoundry) to allow managing edge devices through Kubernetes.
+
```

Comment:
how about add an introduction for `node-resource-manager`(local storage for edge) and `yurtcluster-operator`?

Reply:
I have thought about it but feel they are implementation details that the ToC may not understand easily, especially the yurtcluster-operator.

Reply:
@Fei-Guo OK, but how about add the following capabilities:
- ensure service access traffic(east-west traffic) closed in region(OpenYurt called NodePool)
- pods on the edge node can use InClusterConfig to access kube-apiserver directly without awareness cloud-edge network isolation.